lekce2/8.py

```diff
@@ -0,0 +1,11 @@
+# 8
+import sys
+import math
+
+print("Nacetli jsme prvni cislo: " + sys.argv[1])
+
+vysledek_zaokrouhlovani1 = math.floor(float(sys.argv[1])+0.5)
+print("zaokrouhleno " + str(vysledek_zaokrouhlovani1))
+
+# https://realpython.com/python-f-strings/
+print(f"zaokrouhleno {vysledek_zaokrouhlovani1}")
```

Comment:
Tady jen bacha aby vsechny mely Python >= 3.6. Jestli to instalovaly ted pred zacatkem DA tak snad jo, ale mozna nekdo (hlavne asi na macu, possibly linux jestli nekdo ma) ma nejakou verzi ktera tam byla se systemem, ta by mohla byt starsi. Kdyztak ukaz `print("zaokrouhleno {}".format(vysledek_zaokrouhlovani))` jako alternativu pro ty kterym to nebude fungovat.

Kazdopadne cool, to se jim bude libit.

Reply:
Musej mit >= 3.6 :D at nezlobi! :D